### PR TITLE
Trim patient name

### LINKED ISSUE #195: trim patients names
Screenshot 2024-07-09 at 08 04 06

### PR DESCRIPTION
**Pull request by philipeleandro**
fix #195

### CODE REVIEW

File: db/migrate/20240713155011_update_patients_names.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class UpdatePatientsNames < ActiveRecord::Migration[7.0]
+  def change
+    patients = Patient.all
+
+    patients.each do |patient|
+      patient.update!(name: patient.name)
+    end; nil
```
Comment (leonardobrito) [inline db/migrate/20240713155011_update_patients_names.rb:9]: Do you need the `nil`?

File: app/models/patient.rb [hunk 2]
```diff
@@ -6,7 +6,15 @@ class Patient < ApplicationRecord
 
   validates :name, presence: true
 
+  before_save :remove_name_whitespace
+
   def deletable?
     event_procedures.empty?
   end
+
```
Comment (leonardobrito) [inline app/models/patient.rb:14]: Maybe you can solve it like this also 🤔, with this way you will kinda override the `name setter`. 
Makes sense?
```suggestion
def name=(value)
  self.name = value.strip
end
```

Comment (philipeleandro) [reply app/models/patient.rb]: Totally, I followed your suggestion and change the setter using `super`. Done!

File: db/migrate/20240713155011_update_patients_names.rb (hunk 1 above)
Comment (philipeleandro) [reply db/migrate/20240713155011_update_patients_names.rb]: Actually I don't need it. I ran the migrate and the records of patients weren't displayed. I removed it.

**Review by leonardobrito — APPROVED** [review]

**Pull request merged**